data/remnant missions.txt

```diff
@@ -238,14 +316,39 @@ event "remnant: surveillance end"
 	system "Parca"
 		add fleet "Korath Raid" 10000
 
-
+		
+mission "Remnant: Clean-up Bounty"
```

Comment:
This originally was a bounty targeting some Korath Hunters, a heavy fighter that someone made. I removed the hunter and swapped in Chasers temporarily while they worked on making the 2x sprites and thumbnails, but then they dropped the project, so I guess this mission can simply be scrapped entirely.

Reply:
Thanks to the change to the other Bounty mission, this is now the *only* bounty mission that puts the player up against chasers, so it is now unique. It is also valuable because it gives player an option going forward: Do they want to take the higher paying job against a damaged raider, or do they want to go up against full strength fighters that can swarm them? Different builds will be effective against each of those, and letting people face more than one kind of opponent is valuable. (especially when there's only 3 total ships to oppose right now. People will get more than than enough Raiders and Worldships eventually)